test/edu-benefits/1990/schema/schema.unit.spec.js

```diff
@@ -1,6 +1,6 @@
 import { expect } from 'chai';
 import Ajv from 'ajv';
-import { eduBenefits as schema } from 'vets-json-schema';
```

Comment:
The failure seems to be here, but I'm not clear on how this worked before, and why it wouldn't work with your change.